src/routes/safe/components/Transactions/TxList/Filter/index.tsx

```diff
@@ -25,13 +25,13 @@ import { TX_LIST_EVENTS } from 'src/utils/events/txList'
 
 // Types cannot take computed property names
 const TYPE_FIELD_NAME = 'type'
-const FROM_FIELD_NAME = 'from'
-const TO_FIELD_NAME = 'to'
-const RECIPIENT_FIELD_NAME = 'recipient'
-const HIDDEN_RECIPIENT_FIELD_NAME = '__recipient'
-const AMOUNT_FIELD_NAME = 'amount'
-const TOKEN_ADDRESS_FIELD_NAME = 'tokenAddress'
-const HIDDEN_TOKEN_ADDRESS_FIELD_NAME = '__tokenAddress'
+const FROM_FIELD_NAME = 'execution_date__gte'
+const TO_FIELD_NAME = 'execution_date__lte'
+const RECIPIENT_FIELD_NAME = 'to'
+const HIDDEN_RECIPIENT_FIELD_NAME = '__to'
+const AMOUNT_FIELD_NAME = 'value'
+const TOKEN_ADDRESS_FIELD_NAME = 'token_address'
```

Comment:
I can't find this field name in the transaction documentation. Is it for the GET endpoint `/safes/{address}/multisig-transactions/`?

Reply:
Only incoming: `/safes/{address}/incoming-transfers/`.